new/LIT6842TentaLe.xml

```diff
@@ -0,0 +1,73 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6842TentaLe" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ጥንተ፡ ልደቶሙ፡ ለመነኮሳት፡</title>
+                <title xml:lang="gez" type="normalized" corresp="#t1">Ṭǝnta lǝdatomu lamanakosat</title>
```

Comment:
I think it would be good for quick disambiguition if Dabra Dāmmo would be mentioned in the title